Sources/Array+Remove.swift

```diff
@@ -35,13 +36,13 @@ extension Array {
 
      - parameter object: The object to be removed (if found)
      */
-    mutating func removeObject<T: Equatable>(_ object: T) {
+    public mutating func removeObject<Element: Equatable>(_ object: Element) {
 
         var foundIndex: Int?
```

Comment:
Not specifically related to this PR, but might be worth condensing this in the future to use the `index(of:)` or the  `index(where:)` or  method rather than the `for`-loop.